app/email_batching_test.go

```diff
@@ -314,9 +283,7 @@ func TestRenderBatchedPostGeneric(t *testing.T) {
 	}
 
 	var rendered = th.Server.renderBatchedPost(notification, channel, sender, "http://localhost:8065", "", translateFunc, "en", model.EMAIL_NOTIFICATION_CONTENTS_GENERIC)
-	if strings.Contains(rendered, post.Message) {
-		t.Fatal("Rendered email should not contain post contents when email notification contents type is set to Generic.")
-	}
+	require.Contains(t, post.Message, rendered, "Rendered email should not contain post contents when email notification contents type is set to Generic.")
```

Comment:
I think you have to switch the order of the `post.Message` and `rendered` parameters.